mocks/db.generated.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.39.0. DO NOT EDIT.
```

Comment:
Nitpick: we should consider reverting the changes in the mock files, because it contains only versions change.